locales/en.json

```diff
@@ -1114,5 +1122,96 @@
         "By using the Platform or the Services, you acknowledge that you have read and understood this Policy and Consent to be bound by it and to comply with all applicable laws and regulations."
       ]
     }
+  },
+  "incometax": {
+    "title": "Redirect Income Tax",
+    "p1": {
+      "content": "According to the Fiscal Code ( {0}), taxpayers can direct a percentage of 3.5% of the annual income tax for the support non-profit entities, religious units, as well as fo granting of private scholarships.",
```

Comment:
We should prefer using [slot syntax](https://kazupon.github.io/vue-i18n/guide/interpolation.html#slots-syntax-usage) as it's easier to understand while reading the code because it allows us to label the slot and to use the *template* element in our vue templates.
It's also consistent with the rest of the code 👍

Reply:
Done